src/copyright-license/text/sample.md

```diff
@@ -14,7 +14,9 @@ Permissions. Licensor has obtained from all persons and entities who are, or who
 
 Copyright Notices. Licensee shall ensure that its use of the Work is marked with the appropriate copyright notices specified by Licensor in a reasonably prominent position in the order and manner provided by Licensor. Licensee shall abide by the copyright laws and what are considered to be sound practices for copyright notice provisions in the Territory. Licensee shall not use any copyright notices that conflict with, confuse, or negate the notices Licensor provides and requires hereunder.
 
+{{#clause paymentClause}}
```

Comment:
The tags are in the `sample.md`?

Reply:
Yes. This is the same syntax as the one used in the editor (`markdown_cicero`).